### PR TITLE
install wizard project

### PR DESCRIPTION
**Pull request by MattDHill**

### CODE REVIEW

File: frontend/projects/install-wizard/src/app/pages/home/home.page.ts [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    } catch (e: any) {
+      this.error = e.message
+    } finally {
+      loader.dismiss()
+    }
+  }
+
+  private async presentAlertDanger(logicalname: string, embassyData: boolean) {
+    const message = embassyData
+      ? 'This action is COMPLETELY erase your existing Emabssy data'
```
Comment (elvece) [inline frontend/projects/install-wizard/src/app/pages/home/home.page.ts:79]:
```suggestion
      ? 'This action COMPLETELY erases your existing Embassy data'
```

File: frontend/projects/install-wizard/src/app/pages/home/home.page.ts [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+      : `This action is COMPLETELY erase the disk ${logicalname} and install embassyOS`
```
Comment (elvece) [inline frontend/projects/install-wizard/src/app/pages/home/home.page.ts:80]:
```suggestion
      : `This action COMPLETELY erases the disk ${logicalname} and installs embassyOS`
```

File: frontend/projects/install-wizard/src/app/pages/home/home.page.ts [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+      this.error = e.message
+    } finally {
+      loader.dismiss()
+    }
+  }
+
+  private async presentAlertComplete() {
+    const alert = await this.alertCtrl.create({
+      header: 'Rebooting',
+      message: 'Please wait for Embassy to restart, then refresh this page',
```
Comment (elvece) [inline frontend/projects/install-wizard/src/app/pages/home/home.page.ts:136]:
```suggestion
      message: 'Please wait for embassyOS to restart, then refresh this page',
```

Comment (elvece) [reply frontend/projects/install-wizard/src/app/pages/home/home.page.ts:136]: or "your Embassy"

File: frontend/projects/install-wizard/src/app/services/api/live-api.service.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    })
+  }
+
+  private async rpcRequest<T>(opts: RPCOptions): Promise<T> {
+    const res = await this.http.rpcRequest<T>(opts)
+
+    const rpcRes = res.body
+
+    if (isRpcError(rpcRes)) {
+      throw new RpcError(rpcRes.error)
```
Comment (elvece) [inline frontend/projects/install-wizard/src/app/services/api/live-api.service.ts:41]: will this display in a toast or just in console?

**Review by elvece — CHANGES_REQUESTED** [review]
see copy suggestions

File: frontend/projects/install-wizard/src/app/services/api/live-api.service.ts (hunk 4 above)
Comment (MattDHill) [reply frontend/projects/install-wizard/src/app/services/api/live-api.service.ts:41]: Depends on what is catching it

Comment (MattDHill) [reply frontend/projects/install-wizard/src/app/services/api/live-api.service.ts:41]: Currently will set error = e.message and show on the page

**Review by dr-bonez — APPROVED** [review]

**Review by dr-bonez — APPROVED** [review]

**Pull request merged**